src/pages/editor/ChannelClaimTable.tsx

```diff
@@ -22,10 +29,50 @@ export const ChannelClaimTable = ({ channelClaimList }: ChannelClaimTableProps)
       <TableHead>
         <TableRow sx={{ bgcolor: 'secondary.main' }}>
           <TableCell>{channelType === 'publisher' ? t('common.publisher') : t('common.serial_publication')}</TableCell>
```

Comment:
Noen grunn til at det er lagt til nowrap på andre celler men ikke denne?

![bilde](https://github.com/user-attachments/assets/9e19d512-586d-4374-b348-81ecbb8a375d)
```suggestion
          <TableCell sx={{ textWrap: 'nowrap' }}>{channelType === 'publisher' ? t('common.publisher') : t('common.serial_publication')}</TableCell>
```